node-authorizer/pkg/authorizers/aws/verifier.go

```diff
@@ -55,7 +55,11 @@ func (a *awsNodeVerifier) VerifyIdentity(ctx context.Context) ([]byte, error) {
 	go func() {
 		encoded, err := func() ([]byte, error) {
 			// @step: create a metadata client
-			client := ec2metadata.New(session.New())
+			sess, err := session.NewSession()
+			if err != nil {
+				return []byte{}, err
```

Comment:
Nit (but doesn't matter): returning `nil, err` is the more normal idiom here.  Hopefully the caller checks error before checking the `[]byte`, but an empty slice is more or less equivalent to a nil slice anyway.

Reply:
Ah, yes I was using the other err return on Line 67 as a reference, but it's probably better to use nil regardless.